gae_dashboard/side_by_side_slack_reports.py

```
@@ -86,7 +86,15 @@ def send_to_slack(results_by_team, date):
 
 def main():
     yesterday = datetime.date.today() - datetime.timedelta(days=1)
-    results = fetch_counts_from_bq(yesterday)
+
+    try:
+        results = fetch_counts_from_bq(yesterday)
+    except bq_util.BQException as e:
+        # This can happen if there are no side-by-side experiments
+        # currently in progress.  That's not an error.
+        if 'side_by_side_result_* does not match any table' in str(e):
+            return
+
```

Would we want to re-raise the exception if it doesn't match that string?

---

Ha, good point!